doc/release_notes/release_notes_2.5.rst

```diff
@@ -0,0 +1,189 @@
+.. _release_notes_2.5:
+
+ACRN v2.5 (Jun 2021) DRAFT
+##########################
+
+We are pleased to announce the release of the Project ACRN hypervisor
+version 2.5.
+
+ACRN is a flexible, lightweight reference hypervisor that is built with
+real-time and safety-criticality in mind. It is optimized to streamline
+embedded development through an open-source platform. See the
+:ref:`introduction` introduction for more information.  All project ACRN
```

Comment:
Consider a paragraph break before "All project ACRN source code is maintained...". They you'll have a nice, short description of ACRN as the first paragraph before moving on to source code maintenance etc.